pdns/dnsdist-snmp.cc

```diff
@@ -580,6 +602,7 @@ DNSDistSNMPAgent::DNSDistSNMPAgent(const std::string& name, const std::string& m
   registerGauge64Stat("cpuSysMSec", cpuSysMSecOID, OID_LENGTH(cpuSysMSecOID), &getCPUTimeSystem);
   registerGauge64Stat("fdUsage", fdUsageOID, OID_LENGTH(fdUsageOID), &getOpenFileDescriptors);
   registerGauge64Stat("dynBlockedNMGSize", dynBlockedNMGSizeOID, OID_LENGTH(dynBlockedNMGSizeOID), [](const std::string&) { return g_dynblockNMG.getLocal()->size(); });
+  registerGauge64Stat("securityStatus", securityStatusOID, OID_LENGTH(securityStatusOID), &g_stats.securityStatus);
```

Comment:
Any reason you are not passing a lambda taking a `const std::string&` and returning a `uint64_t`, so basically a `DNSDistStats::statfunction_t`, like is done on the line right above this one?
This would make the need for a new specialization of `registerGauge64Stat()` go away, as well as the need to add the `uint64_t*` type to `DNSDistStats::entry_t`.